sdk/src/main/scala/io/horizen/account/state/ForgerStakeMsgProcessor.scala

```diff
@@ -354,7 +389,7 @@ case class ForgerStakeMsgProcessor(params: NetworkParams) extends NativeSmartCon
       case AddNewStakeCmd => doAddNewStakeCmd(msg, gasView)
       case RemoveStakeCmd => doRemoveStakeCmd(msg, gasView)
       case OpenStakeForgerListCmd => doOpenStakeForgerListCmd(msg, gasView)
-      case opCodeHex => throw new ExecutionRevertedException(s"op code $opCodeHex not supported")
+      case opCodeHex => throw new ExecutionRevertedException(s"op code not supported: $opCodeHex")
```

Comment:
I have a doubt: is this the same exception thrown by EVM when an non existing function is called on a smart contract? It may have sense to have the same message thrown by a real smart contract for a native smart contract.

Reply:
It is not clear to me where this error msg should be found. For the time being I propose to leave it as it is.

Reply:
For me it's ok